android/app/src/androidTest/java/org/ZingoLabs/Zingo/RustFFITest.kt

```diff
@@ -326,6 +326,10 @@ class ExecuteSummariesFromSeed {
         assertThat(initFromSeed.seed).isEqualTo(Seeds.HOSPITAL)
         assertThat(initFromSeed.birthday).isEqualTo(1)
 
+        var price = RustFFI.execute("updatecurrentprice", "")
+        System.out.println("\nPrice:")
+        System.out.println(price)
+
```

Comment:
is there a reason why updatecurrent price is in this test?
i think it would be good to have a separate test for each function in the RustFFI so when a test fails we know exactly which command is failing

Reply:
Well, this is a interesting matter because I thought quite a lot... If we create a test for all the commands that we are using in zingo-mobile, oh man, we need to wait a couple of days to finish the CI. Because the execution of all the `steps` for a single test is really long way. If we want to have a agile CI we need to put several commands in one test, IMO. As you know viewing the logs is pretty straightforward to find out what is the problem.

In this specific case, I wanted to test this command fast, this is why I added to this test... Maybe we can group a few commands in each test that they have some relation between them. Well you are the test framework master, whatever you decide, I agree.

PS: the problem with this command was only for `arm64-v8a`, and now in `dev` is fixed... said that, we can safely close this PR if we want to avoid more confusing in our CI.

Reply:
i thought that might be why. ok i understand, if we can check the logs then faster CI is good. If you change the test name to UpdateCurrentPriceAndSummariesFromSeed we can keep track of which commands are being tested.
maybe we should remove `Execute` from the other test names for consistency but can do this in future PR

Reply:
done!